x-pack/test/security_solution_endpoint/endpoint.config.ts

```diff
@@ -24,5 +24,10 @@ export default async function (ftrConfigProviderContext: FtrConfigProviderContex
     junitReportName: 'X-Pack Endpoint Functional Tests on ESS',
     target: 'ess',
     services,
+    kbnServerArgs: [
+      `--xpack.securitySolution.offeringSettings=${JSON.stringify({
+        disableTimelineSaveTour: true,
+      })}`,
+    ],
```

Comment:
hey @janmonschke,

could you please remove the newly added `offeringSettings` flag from `endpoint.config.ts` and from `serverless.endpoint.config.ts`, and instead add it to `x-pack/test/security_solution_endpoint/config.base.ts`? that is the common config for these FTRs - I believe that would be a better place for this non environment specific flag.

Reply:
@gergoabraham Good point, I did that here: https://github.com/elastic/kibana/pull/169239/commits/8b496838e1a295f70f1cf1ef5fd09846fa6d792a

Reply:
awesome, thank you! 🙇